demo-setup/start.sh

```diff
@@ -0,0 +1,21 @@
+# Script assumes that host system has go-lang installed and polkadex.tar is present in the same directory and config.yaml
```

Comment:
> and polkadex.tar is present in the same directory and config.yaml

What do you mean with the config.yaml?

Reply:
And if we're at it: could you also add the minimum required docker compose version? 1.25 was not new enough.

Reply:
THE config folder ;)

I have added the docker-compose requirements the readme